src/main/scala/tilelink/WidthWidget.scala

```diff
@@ -109,14 +110,16 @@ class TLWidthWidget(innerBeatBytes: Int)(implicit p: Parameters) extends LazyMod
 
       val index  = sel | count
       def helper(idata: UInt, width: Int): UInt = {
-        val mux = Vec.tabulate(ratio) { i => idata((i+1)*outBytes*width-1, i*outBytes*width) }
+        val mux = VecInit.tabulate(ratio) { i => idata((i+1)*outBytes*width-1, i*outBytes*width) }
         mux(index)
       }
 
-      out <> in
+      out.bits := in.bits
```

Comment:
😭 I think out and in are both wires so I had to do this...

Reply:
This is another place where @terpstra's FixChisel3 might help, but OK for now.